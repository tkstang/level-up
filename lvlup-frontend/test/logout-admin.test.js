import LogoutAdmin from '../src/components/Navbar/logout-admin';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('LogoutAdmin should render', () => {
  const store = mockStore({ lvlupApp });
  const logout = shallow(
    <LogoutAdmin store={store} />,
  );
  expect(shallowToJson(logout)).toMatchSnapshot();
});
