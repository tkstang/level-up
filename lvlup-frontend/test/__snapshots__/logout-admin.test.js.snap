exports[`test LogoutAdmin should render 1`] = `
<LogOutAdmin
  loggingOutAdmin={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  } />
`;
